requirements.txt

```diff
@@ -10,3 +9,4 @@ visualdl
 sklearn
 filelock
 nibabel
+opencv-python
```

Comment:
opencv-python is not needed

Reply:
![image](https://user-images.githubusercontent.com/29757093/157159270-fb228306-f8f1-4960-a6a5-4ba2783939e9.png)

The last one is for debug, but I think it's used in visualization

Reply:
You may want to pull the repo, I have deleted the visualize.py